source/jormungandr/jormungandr/instance.py

```diff
@@ -200,6 +169,9 @@ def __init__(
         # Init equipment providers from config
         self.equipment_provider_manager.init_providers(instance_equipment_providers)
 
+        self._get_street_network = streetnetwork_backend_manager.get_street_network
```

Comment:
:smiling_imp:

Reply:
Does it make the code harder to read? I should maybe find an other way...